### PR TITLE
VS Code support

### PR DESCRIPTION
**Pull request by drothlis**

### CODE REVIEW

File: stbt_rig.py [hunk 1]
```diff
@@ -1192,7 +1192,7 @@ def pytest_collect_file(path, parent):
 
     class StbtCollector(pytest.File):
         def collect(self):
-            with open(self.name) as f:
+            with open(self.fspath.strpath) as f:
```
Comment (wmanley) [inline stbt_rig.py:1195]: Yep, this looks safer, and I'd expect it to be fine on Windows too.

File: stbt_rig.py [hunk 2]
```diff
@@ -1249,7 +1249,11 @@ def pytest_sessionstart(session):
         session.stbt_args = args
         resolve_args(session.stbt_args)
 
-        capmanager = session.config.pluginmanager.getplugin('capturemanager')
+        pluginmanager = session.config.pluginmanager
+        if not session.config.option.collectonly:
+            pluginmanager.unregister(name="python")
```
Comment (wmanley) [inline stbt_rig.py:1258]: does this mean that we can remove `'-p', 'no:python',` from our tests? (`test_stbt_rig.py`)

**Review by wmanley — APPROVED** [review]
Great, this looks like it was really fiddly to do.  It's a shame to have to "Disable pytest's built-in python plugin", but c'est la vie.  If there was a better option we'd take it :).

File: stbt_rig.py (hunk 2 above)
Comment (drothlis) [reply stbt_rig.py:1258]: I suppose we could. It was nice to see that `pluginmanager.unregister` doesn't raise if it was already disabled from the command line, as that's what our current PyCharm documentation recommends.

**Pull request merged**